receiver/kafkareceiver/unmarshaller.go

```diff
@@ -27,6 +27,15 @@ type Unmarshaller interface {
 	Encoding() string
 }
 
+// logsUnmarshaller deserializes the message body.
+type logsUnmarshaller interface {
```

Comment:
If this is private means you don't expect users to implement? Also probably we should rename the previous one to TracesUnmarshaller.

Reply:
Thanks, @bogdandrutu. Fixed in https://github.com/open-telemetry/opentelemetry-collector/pull/2944/commits/0cb875fe14ee9e4b8def323d97a84c08d1ffec5a

And actually, there're many methods to be renamed (adding `Traces`). I would like to do this in another PR cuz it's already not a small one.